src/Chapter09/Chapter09.csproj

```diff
@@ -1,11 +1,26 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProductName>Chapter09</ProductName>
+    <ExpectedWarnings>$(ExpectedWarnings)</ExpectedWarnings>
+    <!-- CS0178: Invalid rank specifier: expected ',' or ']' -->
+    <!-- CS1513: } expected -->
+    <!-- CS1514: { expected -->
+    <!-- CS0826: No best type found for implicitly-typed array -->
+    <IgnoredWarnings>$(IgnoredWarnings),CS0178,CS1513,CS1514,CS0826</IgnoredWarnings>
```

Comment:
Do we need to exclude these warnings? I would expect that we probably want to suppress them on a file by file basis rather than at the project level.